library/render-test-config.sh

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
```

Comment:
afaict this script just writes the azure secrets locally, which should not be needed anymore given the shift to the DefaultAzureCredential.